src/Adapters.Tests/NLogTarget.Tests.Shared/NLogTargetTests.cs

```diff
@@ -232,12 +232,31 @@ public void TraceAreEnqueuedInChannelAndContainExceptionMessage()
                 aiLogger.Debug(exception, "testing exception scenario");
             }
 
-            var telemetry = (ExceptionTelemetry)this.adapterHelper.Channel.SentItems.FirstOrDefault();
+            var telemetry = (ExceptionTelemetry)this.adapterHelper.Channel.SentItems.First();
             Assert.AreEqual(expectedException.Message, telemetry.Exception.Message);
         }
 
         [TestMethod]
         [TestCategory("NLogTarget")]
+        public void CustomMessageIsAddedToExceptionTelemetryCustomProperties()
+        {
+            Logger aiLogger = this.CreateTargetWithGivenInstrumentationKey();
+            
+            try
+            {
+                throw new Exception("Test logging exception");
```

Comment:
Do you have to throw it in a try catch or can you just create it and pass it to the aiLogger.Debug(...) call without the extra lines of code?